### DISCUSSION TITLE
Could i use Unicorn to load components after page load

### DISCUSSION THREAD

**Discussion by Matthijz98** [django-commons/django-unicorn#503] [Q&A]
I have a recemented products component on a page but the query for that takes a while sometimes if not cached.
Could i use Unicorn to load it later so have a empty component in the loading stage and than wait on the sql to finish before loading?

I thought of doing it with the mount() method. But i was wondering if there are other ways i could takel this.

**Comment by adamghill** [comment]
Yes! That would definitely work. Using `mount()` would probably be the easiest way. Let me know how it goes! 👍

**Comment by Matthijz98** [comment]
I gave it a try but the hole pages gets delayed until mount is done.
When i try this the hole page load takes 10 seconds longer
```
class TestView(UnicornView):
    def mount(self):
        time.sleep(10)
```
